installation/generate_helm_values.py

```diff
@@ -302,11 +302,19 @@ def iter_helm_values() -> collections.abc.Generator[tuple[str, dict], None, None
         delivery_db_backup_enabled = 'deliveryDbBackup' in extensions
         freshclam_enabled = 'clamav' in extensions
 
+        delivery_service_cfg = cfg_set.delivery_service()
+
+        # inject rescoring cfg from delivery-service cfg to avoid duplication
+        rescoring_cfg = delivery_service_cfg.features_cfg().get('rescoring')
+
         configuration = {
             'scanConfigurations': [
                 {
                     'name': normalize_name(extension_cfg.name()),
-                    'spec': extension_cfg.raw,
+                    'spec': dict(
+                        **extension_cfg.raw,
+                        **({'rescoring': rescoring_cfg} if rescoring_cfg else {})
```

Comment:
I think this is not too intuitive to read, what about setting `{}` as default when retrieving `rescoring_cfg` and drop the condition here?